src/demo-app/datepicker/custom-header.scss

```diff
@@ -0,0 +1,10 @@
+.custom-header {
+  padding: 1.5em;
```

Comment:
Changing the buttons to `mat-icon-button` threw off the spacing I originally gave you, can you change these rules to:  

```
padding: 1em 1.5em;
display: flex;
align-items: center;
```

Reply:
done in b6fc003e8919b4a898c4bc8fe0a23f4eecc2e43a

Reply:
It looks nice now, thanks for the CSS!